test_api_crawler assumes it is run in the source directory

#### ALL software version info (bokeh, python, notebook, OS, browser, any other relevant packages)
* bokeh-0.12.4
* Python 2.7.13
* Fedora 25

#### Description of expected behaviour and the observed behaviour
`test_api_crawler`, when run on the installed package, fails because it assumes it's being run in the source directory. But when testing the installed package, it's quite likely that you aren't in the source directory.

#### Complete, minimal, self-contained example code that reproduces the issue
$ pip install --user bokeh && py.test --pyargs bokeh -k test_get_crawl_dict

#### Stack traceback and/or browser JavaScript console output
```
___________________________________________________________ test_get_crawl_dict ____________________________________________________________

test_crawler = <bokeh.util.api_crawler.APICrawler object at 0x7fa9edb97610>

    def test_get_crawl_dict(test_crawler):
        crawl_dict = test_crawler.get_crawl_dict()
>       assert crawl_dict
E       assert {}

.local/lib/python2.7/site-packages/bokeh/util/tests/test_api_crawler.py:37: AssertionError
```

--- a/bokeh/util/tests/test_api_crawler.py
+++ b/bokeh/util/tests/test_api_crawler.py
@@ -1,5 +1,6 @@
 from __future__ import absolute_import
 import ast
+import os
 import pytest
 
 
@@ -28,7 +29,9 @@ def sample_function(self):
 
 @pytest.fixture
 def test_crawler():
-    return api_crawler("bokeh")
+    path = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                        os.pardir, os.pardir))
+    return api_crawler(path)
 
 
 def test_get_crawl_dict(test_crawler):

